### ISSUE TITLE
Disable some dates in the date picker

### ISSUE THREAD

**Issue by AITSco** [xgfe/react-native-datepicker#163]
Dears,

I looking for disable some dates in the date picker to be dimmed ( disable weekends for example) but i couldn't find property to do that .would you please help

Thanks

**Comment by chapeljuice** [comment]
I'm looking for this feature as well. @AITSco did you happen to find anything or have any luck?

**Comment by AITSco** [comment]
Unfortunately no. kindly let me know if you have solution

Thanks

**Comment by chapeljuice** [comment]
@AITSco I'm actually thinking of jumping ship and removing this module from my app, and going with `react-native-calendars`, FYI. We'll see!

**Comment by pandiyanmadhan** [comment]
How to disable weekdays like saturday and sunday only.

**Comment by Muhammad-Hasham-Khalid** [comment]
Yes this feature is needed so that we can disable some dates and user will not be able to pick from them.
